fastlane/Deliverfile

```diff
@@ -0,0 +1,18 @@
+# The Deliverfile allows you to store various App Store Connect metadata
+# For more information, check out the docs
+# https://docs.fastlane.tools/actions/deliver/
+
+app_version("8.1.7")
```

Comment:
How will this work with the `fastlane/VERSION`? Not sure whether introducing this `app_version` here may induce the fastlane action to pick it as priority